src/cuttlefish.erl

```diff
@@ -135,6 +139,8 @@ otp_test() ->
     ?assert(otp("R16", "R16A")),
     ?assert(not(otp("R16B01", "R16A"))),
     ?assert(otp("R16A", "R16A")),
+    ?assert(otp("R16A", "17.4")),
+    ?assert(not(otp("18.2.4", "17.4"))),
```

Comment:
Additional test cases should include the more basic versions, too. E.g. `otp("R16", "17")`, `not(otp("17", "R16"))`, etc.